opm/grid/polyhedralgrid/grid.hh

```diff
@@ -972,60 +973,41 @@ namespace Dune
     {
       const int codim = EntitySeed :: codimension;
       const int index = seed.index();
-      switch (codim)
-      {
-        case 0:
-          {
-            return cellVertices_[ index ].size();
-          }
-        case 1:
-          {
-            //return grid_.cell_facepos[ index+1 ] - grid_.cell_facepos[ index ];
-            return grid_.face_nodepos[ index+1 ] - grid_.face_nodepos[ index ];
-          }
-        case dim:
-          {
-            return 1;
-          }
-        default:
-          {
-            return 0;
-          }
-      }
+      if (codim==0)
+        return cellVertices_[ index ].size();
+      if (codim==1)
+        return grid_.face_nodepos[ index+1 ] - grid_.face_nodepos[ index ];
+      if (codim==dim)
+         return 1;
+      return 0;
```

Comment:
This change is necessary to make PolyhedralGrid work for 1D grids (then dim = 1, and two cases in the switch are equal). In this case faces and nodes have the same co-dimension. With the current change we always return the face properties for codim=1. If the grid dimension is 1, should we rather return the node properties? I'm not sure if it matters as there is (always?) a one-to-one mapping between faces and nodes in 1D.